piu/cli.py

```diff
@@ -200,6 +199,12 @@ def cli(ctx, config_file):
     ctx.obj = config_file
 
 
+def lookup_instance(region, ip_address):
```

Comment:
Please move the helper function between class and cli definition. Don't mix up functions and cli-functions